internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -116,6 +116,30 @@ func convertContainerHealthCheck(hc manifest.ContainerHealthCheck) *template.Con
 	}
 }
 
+func convertHostedZone(m manifest.RoutingRuleConfiguration) template.HostedZoneToAliases {
+	hostedZoneToAliases := make(map[string][]string)
```

Comment:
nit: (I typically use `For` when creating maps. I first saw this from Martin Fowler's refactor book)
```suggestion
func convertHostedZone(m manifest.RoutingRuleConfiguration) template.AliasesForHostedZone {
	aliasesFor := make(map[string][]string)
```